README.markdown

```diff
@@ -1056,6 +1056,20 @@ In larger expressions, optional parentheses can sometimes make code read more cl
 let playerMark = (player == current ? "X" : "O")
 ```
 
+## NSLocalizedStrings
```

Comment:
Added what we currently do for `NSLocalizedStrings`. Would really appreciate input if you've localized apps before and have ideas for better practices

Reply:
Nice addition Peter.

I have experience working with localization (in fact this is the first non-localized project I work with) and for now using localized strings is good 👍 


There are 2 nice additions we could use in the future:
1. Pluralization: things like.. "1 day", "2 days" (where 1 and 2 are variables). Xcode has support for it but we can add it when we face that problem.
2. Use a tool like swiftgen. I don't think we need that now. But it is really useful. Basically generates an enum from the localization.strings file, and you can substitute `NSLocalizedString("dismiss", comment: "dismiss")` by `translate(.dismiss)`. But this requires an external translator (or translation service) to create the localization.strings for us.

Reply:
Sounds good 👍